File: nnt/lib/core/logger.dart
part of nnt.core;

// 基础的日志管理器
class Logger {
  log(String msg) {
    print(msg);
  }

  warn(String msg) {
    print(msg);
  }

  fatal(String msg) {
    print(msg);
  }
}

final Logger logger = new Logger();

File: impl/android/src/main/kotlin/com/nnt/core/Logger.kt
package com.nnt.core;

import android.util.Log

class Logger {
    companion object {
        fun log(msg: String) {
            Log.d("nnt", msg)
        }

        fun warn(msg: String) {
            Log.w("nnt", msg)
        }

        fun fatal(msg: String) {
            Log.e("nnt", msg)
        }
    }
}